examples/mongo/README.md

```diff
@@ -0,0 +1,25 @@
+# Mongo Metrics
+
+The [`mongo_3.rb`](mongo_3.rb) script generates metrics from the `mongotop` and `mongostat` commands for a Mongo 3.0 database. The following metrics will be output for every collection in your system:
```

Comment:
I'd actually be more pro Mongo 2, Mongo 3 support was an accident of me developing while having Mongo 3 locally, then realizing Mongo 2 tools don't support the JSON output mode. A Mongo 2 script would have a lot more text parsing, which is nbd, but would require doing the thing + testing.